ui/css/treeherder-global.css

```diff
@@ -306,7 +306,7 @@ input[type='checkbox'] {
 }
 
 .bg-lightgray {
-  background-color: lightgray;
+  background-color: whitesmoke;
```

Comment:
This lightgray is fine according to the color contrast checker: https://webaim.org/resources/contrastchecker/ so lets keep it (plus it looks better by providing contrast against the white background).